frontstage/controllers/party_controller.py

```diff
@@ -233,6 +233,17 @@ def get_respondent_enrolments(party_id):
                 }
 
 
+def get_respondent_enrolments_for_known_collex(enrolment_data, cache_collex):
+    """remove any enrolments that are for surveys not in collex
```

Comment:
This docstring is missing some punctuation or something.  I don't really understand it

Reply:
There was another line in the docstring. Updated for clarity